mercury/src/internal/pack/decode.rs

```diff
@@ -750,10 +750,10 @@ mod tests {
                 cnt += 1; //use entry here
             }
             tracing::info!("Received: {}", cnt);
-            count_c.store(cnt, Ordering::Relaxed);
+            count_c.store(cnt, Ordering::Release);
         }).await.unwrap();
         let p = pack.await.unwrap();
-        assert_eq!(count.load(Ordering::Relaxed), p.number);
+        assert_eq!(count.load(Ordering::Acquire), p.number);
```

Comment:
Here, `Ordering::Relaxed` maybe okay? `await` is used, so the sub-thread (`spawn_blocking`) has already over when `count.load()`.

Reply:
Indeed. But it's in a test and only called once, don't be so serious :)